### ISSUE TITLE
Blank Footer

### ISSUE THREAD

**Issue by odenypeter** [open-build/eLearning#19]
The footer is currently blank. Can we have some info.

**Comment by andrewtpham** [comment]
I assume placing the OpenBuild name and a link to the open source license we are using. Greg?

**Comment by glind** [comment]
Yeah agreed.  Let's put the OpenBuild name and a link to the license in GitHub.  Also let's link to the Fork of the original eLearning repo with a "based on eLearning by ... "

**Comment by odenypeter** [comment]
I fully agree.
